apps/src/templates/codeReviewGroups/AssignedStudentsPanel.jsx

```diff
@@ -28,7 +28,7 @@ export default function AssignedStudentsPanel({
           onClick={onCreateGroupClick}
           icon={<FontAwesome icon="plus" className="fa" />}
           text={i18n.createGroup()}
-          style={styles.button}
+          inlineStyle={styles.button}
```

Comment:
Keep this prop isn't ideal, but I didn't want to get into refactoring these components into SCSS as well.

Reply:
Yeah, that seems reasonable to me 👍